.cicd/scripts/srw_ftest.sh

```diff
@@ -53,35 +53,32 @@ pwd
 #     "Run the Workflow Using Stand-Alone Scripts".
 echo "BRANCH=${BRANCH}"
 
-# Set the ACCOUNT to use for this PLATFORM ...
-sed "s|^  ACCOUNT: \"\"|  ACCOUNT: \"${SRW_PROJECT}\"|1" -i ush/config_defaults.yaml
-sed "s|hera|${platform,,}|1" ush/config.community.yaml | sed "s|an_account|${SRW_PROJECT}|1" > ush/config.yaml
-
-# Set directory paths ...
-export EXPTDIR=${workspace}/expt_dirs/test_community
-echo "EXPTDIR=${EXPTDIR}"
-sed "s|^workflow:|workflow:\n  EXPT_BASEDIR: ${workspace}/expt_dirs|1" -i ush/config.yaml
-sed "s|^workflow:|workflow:\n  EXEC_SUBDIR: ${workspace}/install_${SRW_COMPILER}/exec|1" -i ush/config.yaml
-
-# Decrease forecast length since we are running all the steps
-sed "s|^  FCST_LEN_HRS: 12|  FCST_LEN_HRS: 6|g" -i ush/config.yaml
-
-# Update compiler 
-sed "s|^  COMPILER: intel|  COMPILER: ${SRW_COMPILER}|g" -i ush/config.yaml
+# Set test settings for the platform and account
+EXPTDIR=${workspace}/expt_dirs/test_community
 
 # DATA_LOCATION differs on each platform ... find it.
-export DATA_LOCATION=$(grep TEST_EXTRN_MDL_SOURCE_BASEDIR ${workspace}/ush/machine/${platform,,}.yaml | awk '{printf "%s", $2}')
+DATA_LOCATION=$(grep TEST_EXTRN_MDL_SOURCE_BASEDIR ${workspace}/ush/machine/${platform,,}.yaml | awk '{printf "%s", $2}')
 echo "DATA_LOCATION=${DATA_LOCATION}"
 
-# Configure a default test ...
-sed "s|^task_get_extrn_ics:|task_get_extrn_ics:\n  EXTRN_MDL_SOURCE_BASEDIR_ICS: ${DATA_LOCATION}/FV3GFS/grib2/2019061518|1" -i ush/config.yaml
-sed "s|^task_get_extrn_lbcs:|task_get_extrn_lbcs:\n  EXTRN_MDL_SOURCE_BASEDIR_LBCS: ${DATA_LOCATION}/FV3GFS/grib2/2019061518|1" -i ush/config.yaml
-
-# Use staged data for HPSS supported machines
-sed 's|^platform:|platform:\n  EXTRN_MDL_DATA_STORES: disk|g' -i ush/config.yaml
-
-# Set OMP_NUM_THREADS_RUN_FCST to 1 in config.yaml
-sed 's|^task_run_fcst:|task_run_fcst:\n  OMP_NUM_THREADS_RUN_FCST: 1|1' -i ush/config.yaml
+user_settings="
+user:
+  ACCOUNT: ${SRW_PROJECT}
+  MACHINE: ${platform,,}
+workflow:
+  EXPT_BASEDIR: ${workspace}/expt_dirs
+  EXEC_SUBDIR: ${workspace}/install_${SRW_COMPILER}/exec
+  FCST_LEN_HRS: 6
+  COMPILER: ${SRW_COMPILER}
+task_get_extrn_ics:
+  envvars:
+    EXTRN_MDL_SOURCE_BASEDIR_ICS: ${DATA_LOCATION}/FV3GFS/grib2/2019061518
+task_get_extrn_lbcs:
+  envvars:
+    EXTRN_MDL_SOURCE_BASEDIR_LBCS: ${DATA_LOCATION}/FV3GFS/grib2/2019061518
+task_run_fcst:
+  envvars:
+    OMP_NUM_THREADS_RUN_FCST: 1
+"
```

Comment:
Because the structure of config_defaults.yaml changed, these sed commands no longer work. Instead of relying on users' understanding of even more complex sed mods, and because it's likely dangerous to rely on the first occurrence of the string to change the item, I've incorporated uwtools in here, as well. This structure makes it clearer to modify when we have this sort of issue in the future.